app/views/csv_exports/new_users.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Request a CSV Report</h1>
+
+<section>
+  <div class="csv-users-filters clearfix">
+    <%= form_tag users_csv_exports_path(format: :csv), method: :get, class: "form-horizontal" do %>
+      <div><strong>User Permission Reports</strong></div>
+      <div class="form-group">
+        <%= label_tag :inherited, "Show inherited project permissions", class: "col-lg-2 control-label" %>
+        <div class="col-lg-2">
+          <%= select_tag :inherited, options_for_select([['No', false], ['Yes', true]], false), class: "form-control" %>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= label_tag :deleted, "Include deleted users", class: "col-lg-2 control-label" %>
+        <div class="col-lg-2">
+          <%= select_tag :deleted, options_for_select([['No', false], ['Yes', true]], false), class: "form-control" %>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= label_tag :project_id, "Project", class: "col-lg-2 control-label" %>
+        <% projects = [['', nil]] + Project.pluck(:name, :id) %>
+        <div class="col-lg-4">
+          <%= select_tag(:project_id, options_for_select(projects), class: "form-control") %>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= label_tag :user_id, "User", class: "col-lg-2 control-label" %>
+        <% users = [['', nil]] + User.pluck(:name, :id) +
+            User.unscoped.where("deleted_at IS NOT NULL").pluck(:name, :id).map { |u| ["(deleted) #{u[0]}", u[1]] } %>
```

Comment:
does this need to be unscoped? Probably another good place to use `User.with_deleted { User.pluck(:name, :id) }`